shared/macros.jinja

```diff
@@ -513,7 +513,11 @@ ocil_clause: "{{{ sebool }}} is not enabled"
     <pre>$ sysctl {{{ sysctl }}}</pre>
     The output of the command should indicate a value of <code>{{{ value }}}</code>.
     If this value is not the default value, investigate how it could have been
-    adjusted at runtime, and verify it is not set improperly in <tt>/etc/sysctl.conf</tt>.
+    adjusted at runtime, and verify it is not set improperly by running the
+    following command to search for such lines in all files in <tt>/etc/sysctl.d</tt>
+    and the deprecated <code>/etc/sysctl.conf</code>:
+    <pre>$ grep -r {{{ sysctl }}} /etc/sysctl.conf /etc/sysctl.d</pre>
+
```

Comment:
Sorry for being nitpicky, but this is quite hard to follow sentence. Can you rephrase it? For example
"...adjusted at runtime, and verify it is not set improperly. This has to be checked in all files in <tt>/etc/sysctl.d</tt> and the deprecated <code>/etc/sysctl.conf</code>. Following command will do just that:"

Reply:
sure.... done.